packages/foreman/tfm/tfm.spec

```diff
@@ -305,6 +306,9 @@ selinuxenabled && load_policy || :
 %{_root_sysconfdir}/rpm/macros.%{scl_name}-scldevel
 
 %changelog
+* Mon Nov 18 2019 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
```

Comment:
This is failing linting due to missing version

Reply:
I wonder how obal messed up there. Fixing manually.